packages/teams-js/src/public/appNotification.ts

```diff
@@ -0,0 +1,98 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the appNotifications specific part of the SDK
+ */
+export namespace appNotification {
+  /**
+   * Data structure to represent appNotification information
+   */
+  export interface NotificationDisplayParam {
+    /**The heading of the notiification to be delivered,it would have a maximum string length of 20*/
+    title: string;
+    /**the body of information which would be displayed in the notification box,it would have a maximum string length of 40*/
+    Content: string;
+    /**this would represent an icon that can be displayed on the notification*/
+    icon?: HTMLImageElement;
+    /**This would specify how long a notification would be displayed on the screen for*/
+    itemId: number;
+    /**a url link to the page in which the notification would direct the user to.*/
+    url?: string;
```

Comment:
>url [](http://example.com/codeflow?start=4&length=3)

Can you use a more specific name for this that describes its usage, e.g. `notificationActionUrl`